subprojects/docs/src/docs/release/notes.md

```diff
@@ -302,22 +328,22 @@ The following are the features that have been promoted in this Gradle release.
 ### Example promoted
 -->
 
-### TestKit becomes public feature
-
-TestKit was first introduced in Gradle 2.6 to support developers with writing and executing functional tests for plugins. In the course of the Gradle 2.x releases, a lot of new functionality was added. This version of Gradle removes the incubating status and makes TestKit a public feature.
-
-### `CompileOptions.annotationProcessorPath` property
-
-The `CompileOptions.annotationProcessorPath` property has been promoted and is now stable.
-
-### Build cache and task output caching
+### Build cache and task output caching marked stable
 
 The [build cache](userguide/build_cache.html) and [task output caching](userguide/build_cache.html#sec:task_output_caching) were first introduced in Gradle 3.5.
 They are used in production by different teams, including the Gradle team itself, with great results.
 As of Gradle 4.6, the build cache and task output caching are no longer incubating and considered public features.
 
 Note that the SPI to [implement your own build cache service](userguide/build_cache.html#sec:build_cache_implement) stays incubating.
 
+### TestKit marked stable
+
+[TestKit](userguide/test_kit.html) was first introduced in Gradle 2.6 to support developers with writing and executing functional tests for plugins. In the course of the Gradle 2.x releases, a lot of new functionality was added. This version of Gradle removes the incubating status and makes TestKit a stable feature.
+
+### `CompileOptions.annotationProcessorPath` property
```

Comment:
We have now twice "... marked stable" and header only referring to a property name. Shouldn't this be somewhat similar? Did you choose to change it to the way it is since we are referring to two features and one method?

Reply:
You're right, we should add "marked stable" I think.